### DISCUSSION TITLE
How to adjust the height of "Switch"?

### DISCUSSION THREAD

**Discussion by HuaxiaGoodBoy** [flet-dev/flet#3996] [Q&A] [answered]
Although I use different height settings for mutiple switch controls in constractor, their heights all look the same.

![2fd0b9b7fa8404b6db15b850d2ba3e1](https://github.com/user-attachments/assets/eb518067-b813-4443-8ae1-f08740d1dda1)

**Comment by ndonkoHenri** [comment, marked as answer]
Thanks for reporting. Please sub to https://github.com/flet-dev/flet/issues/3997
